src/bash/report_campaigns/aggregate_campaign_tier

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+if [ $# -eq 0 ] || [ "$1" == "-h" ] || [ "$1" == "-help" ] || [ "$1" == "--help" ]; then
+    echo "Usage: ./"$(basename "$0")" <date> [location]"
+    echo "Example: ./"$(basename "$0")" 20170228 /cms/users/$USER/campaign_tier"
+    echo "<date> = PhEDEx date in the following format: YYYYMMDD."
+    echo "[location] = hdfs location of resulting files. Default: /cms/users/$USER/campaign_tier"
+    exit 0
+fi
+
+# Parse date argument
+date=''
+
+if [[ -n $1 ]]
+then
+    date=$1
+fi
+
+date_length=${#date}
+if [[ $date_length != 8 ]]
+then
+    echo 'Invalid date. Correct format: YYYYMMDD. Example: 20170228'
+    exit
+fi
+
+echo 'Aggregating for date: '$date
+
+location=/cms/users/$USER/campaign_tier
+
+if [[ -n $2 ]]
+then
+    location=$2
+fi
+
+echo 'Results path: '$location
+
+# Results will be put here
+hdir=hdfs://$location
```

Comment:
The location should be provided in a form of /path or hdfs:///path Please adjust code/usage to handle this case. If somehow user pass hdfs:///path it will fail here since you already defined a prefix.